LanguagePlatform/Sdl.SDK.LanguagePlatform.Samples.TmLookup/Properties/AssemblyInfo.cs

```diff
@@ -9,7 +9,7 @@
 [assembly: AssemblyConfiguration("")]
 [assembly: AssemblyCompany("")]
 [assembly: AssemblyProduct("Sdl.SDK.LanguagePlatform.Samples.TmLookup")]
-[assembly: AssemblyCopyright("Copyright ©  2023")]
+[assembly: AssemblyCopyright("Copyright ©  2024")]

```

Comment:
Reminder: We'll need to review the copyright for all of these in a separate PR

Example:
[assembly: AssemblyCompany("SDL Limited as part of the RWS Holdings Plc group of companies")]
[assembly: AssemblyCopyright("Copyright © 2011 - 2022 SDL Limited as part of the RWS Holdings Plc group of companies (\"RWS Group\").")]

Reply:
Done